schema/home/fetch.js

```diff
@@ -22,6 +23,12 @@ export const featuredFair = () => {
   });
 };
 
+export const activeSaleArtworks = (accessToken) => {
+  return gravity.with(accessToken)('me/lot_standings').then((results) => {
```

Comment:
Oh, `live` refers to whether or not the sale is _open for bidding_, not whether it's a live auction integration. If you don't pass `live=true` you'll also get bids on sales that are closed.